sdk/python/feast/registry_server.py

```diff
@@ -580,6 +582,17 @@ def Refresh(self, request, context):
     def Proto(self, request, context):
         return self.proxied_registry.proto()
 
+    def SetDecisionStrategy(
```

Comment:
I'm wondering what permission we should enforce here, or can we allow any user to change this global setting for all the users?

You can keep as-is for now, but we'll change it later when we address the "Intra-service communication" ticket.